PWGCF/EbyEFluctuations/Tasks/MeanPtFlucIdentified.cxx

```diff
@@ -638,10 +660,28 @@ struct meanPtFlucId {
     hist.fill(HIST("QA/Kaon/innerParam/before/h2_TpcTofNsigma"), track.tpcNSigmaKa(), track.tofNSigmaKa());
   }
 
+  // Moments Calculation:
+  void moments(double pt, double& Q1, double& Q2, double& Q3, double& Q4)
+  {
+    Q1 += pt;
+    Q2 += pt * pt;
+    Q3 += pt * pt * pt;
+    Q4 += pt * pt * pt * pt;
+  }
+
   // Fill after PID cut QA hist:
   template <int mode, typename T>
-  void FillIdParticleQAHistos(T const& track, double rap, double nSigmaTPC, double nSigmaTOF)
+  void FillIdParticleQAHistos(T const& track, double rap, double nSigmaTPC, double nSigmaTOF, int& N, double& Q1, double& Q2, double& Q3, double& Q4)
   {
+    N++;
+    moments(track.pt(), Q1, Q2, Q3, Q4);
```

Comment:
In here, an apparently filling QA histograms, an update of global variables is carried out
I would suggest not doing such unexplicit 'hidden' updates
They are really error prone, highly not readable, and compromises functionality evolution